migrations/Version202107271111381465_qtiItemPci.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+declare(strict_types=1);
+
+namespace oat\qtiItemPci\migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use oat\tao\scripts\tools\migrations\AbstractMigration;
+use oat\tao\scripts\tools\accessControl\SetRolesAccess;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version202107271111381465_qtiItemPci extends AbstractMigration
+{
+
+    private const CONFIG = [
+        SetRolesAccess::CONFIG_RULES => [
+            'http://www.tao.lu/Ontologies/TAOItem.rdf#ItemContentCreatorRole' => [
```

Comment:
Can we use the constant for `http://www.tao.lu/Ontologies/TAOItem.rdf#ItemContentCreatorRole`?

Reply:
Can I use the constant defined on taoItems/models/classes/user/TaoItemsRoles.php

Reply:
I think yes, the place where you have this constant.

This extension depends on itemqti that depends on items extension: https://github.com/oat-sa/extension-tao-itemqti-pci/blob/master/composer.json#L27

Reply:
Done